client/src/styles/chat.css

```diff
@@ -0,0 +1,72 @@
+#root {
```

Comment:
This file doesn't seem to be imported anywhere, can it be removed?

Reply:
Removed chat.css file.